packages/x-components/src/x-modules/extra-params/components/renderless-extra-param.vue

```diff
@@ -75,7 +75,7 @@ default value of it.
 ```vue
 <template>
   <RenderlessExtraParam #default="{ value, updateValue }" name="store">
-    <BaseDropdown @change="updateValue" :value="value" :items="items" />
+    <BaseDropdown @update:modelValue="updateValue" :modelValue="value" :items="items" />
```

Comment:
We definitely should mark this as breaking-change. I bet there are a few setups using this approach

Reply:
I agree. Once the PR is approved, I will mark a commit as breaking change